packages/ui-components/src/components/table/Table.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { MouseEventHandler, ReactNode } from "react";
+
+export interface TableProps {
+	columns: string[];
```

Comment:
Should be named `columnHeaders` instead?

```suggestion
	columnHeaders: string[];
```